.golangci.yaml

```diff
@@ -0,0 +1,33 @@
+# See https://github.com/golangci/golangci-lint/blob/master/.golangci.example.yml
+run:
+  tests: false
+  
+  timeout: 5m
+
+  build-tags:
+    - aws
+
+  skip-dirs:
+    - tools
+    - website
+    - hack
+    - charts
+    - designs
+
+linters:
+  enable:
+    - asciicheck
+    - errorlint
+    - gosec
+    - revive
+    - stylecheck
+    - tparallel
+    - unconvert
+    - unparam
+  disable:
+    - errcheck
+    - prealloc
```

Comment:
This is the only field that diverges from kn IIUC

Reply:
See my other comment about why `prealloc` isn't a good idea.